packages/editor/src/client/base.tsx

```diff
@@ -214,6 +214,7 @@ export function Base({children}: BaseProps) {
                   trigger="click"
                   renderTitle={() => (
                     <Icon
+                      className="language-selector"
```

Comment:
What's the advantage of having the icon blue only on hover? For consistency, can we have the same styling on the settings icon?